projects/deja-js/component/editor/deja-editor.component.ts

```diff
@@ -158,14 +158,15 @@ export class DejaEditorComponent extends Destroy implements OnChanges, AfterView
      * On component view init
      */
     public ngAfterViewInit() {
-        this._initializer.initDejaEditorLib().then(() => {
+        from(this._initializer.initDejaEditorLib()).pipe(
+            take(1),
+            takeUntil(this.destroyed$)
```

Comment:
tap(() => this.ckeditorInit
delay(0)
takeUntil
subscribe(() => this._change

Reply:
C'est corrigé, merci